From 9e491db7d493c72fb1dde47f4a5b240fe950656d Mon Sep 17 00:00:00 2001
From: Ryan Volz <ryan.volz@gmail.com>
Date: Tue, 15 Sep 2020 13:44:19 -0400
Subject: [PATCH] runtime: lib: Include missing <algorithm> header.

This fixes compilation with MSVC on conda-forge, although why this
wasn't needed earlier is a mystery.
---
 gnuradio-runtime/lib/io_signature.cc | 1 +
 gr-fec/lib/polar_common.cc           | 1 +
 2 files changed, 2 insertions(+)

diff --git a/gnuradio-runtime/lib/io_signature.cc b/gnuradio-runtime/lib/io_signature.cc
index e5fd2bdd7..b330e56c3 100644
--- a/gnuradio-runtime/lib/io_signature.cc
+++ b/gnuradio-runtime/lib/io_signature.cc
@@ -13,6 +13,7 @@
 #endif
 
 #include <gnuradio/io_signature.h>
+#include <algorithm>
 #include <iostream>
 #include <stdexcept>
 
diff --git a/gr-fec/lib/polar_common.cc b/gr-fec/lib/polar_common.cc
index 6df894766..1b2a13137 100644
--- a/gr-fec/lib/polar_common.cc
+++ b/gr-fec/lib/polar_common.cc
@@ -19,6 +19,7 @@
 #include <gnuradio/blocks/pack_k_bits.h>
 #include <gnuradio/blocks/unpack_k_bits.h>
 
+#include <algorithm>
 #include <cmath>
 #include <iostream>
 #include <stdexcept>
-- 
2.20.1

